StarRailBattleSim/src/lightcones/destruction/SomethingIrreplaceable.java

```diff
@@ -0,0 +1,28 @@
+package lightcones.destruction;
+
+import characters.AbstractCharacter;
+import enemies.AbstractEnemy;
+import lightcones.AbstractLightcone;
+import powers.PermPower;
+import powers.PowerStat;
+import powers.TempPower;
+
+public class SomethingIrreplaceable extends AbstractLightcone {
+
+    public SomethingIrreplaceable(AbstractCharacter owner) {
+        super(1164, 582, 397, owner);
+    }
+
+    @Override
+    public void onEquip() {
+        this.owner.addPower(PermPower.create(PowerStat.ATK_PERCENT, 24, "Something Irreplaceable ATK Boost"));
+    }
+
+    @Override
+    public void onAttacked(AbstractEnemy enemy) {
+        // TODO: Restore HP
+        TempPower power = TempPower.create(PowerStat.DAMAGE_BONUS, 24, 1, "Something Irreplaceable Damage Bonus");
+        power.justApplied = false; // Removes on next turn start
```

Comment:
I'd only looked at the onEndTurn code, but yes it is. The power would be kept during one turn, I'd have to add something to start to decrease it at the start of the turn. 
Or if it i already there? And I just looked over it

Reply:
I tested the LC on a private server (since I don't have it on live) and it looks like it's removed at the end of turn, so normal turn-based power timing should work. (idk why the LC says ` until the end of their next turn` instead of just saying `lasts for 1 turn`.